ontology/disease_syn_enrich.py

```diff
@@ -0,0 +1,188 @@
+#script for enriching disease terms with synonyms
+import os
+import requests
+import json
+import re
+import pandas as pd
+import mysql.connector as mysql
+import joblib
+import numpy as np
+from dotenv import load_dotenv
+
+#loading SQL table
+load_dotenv('.env')
+
+user = os.environ.get("mysql_user")
+pw=os.environ.get("mysql_pwd")
+db=os.environ.get("mysql_db")
+host= "localhost"
+
+conn = mysql.connect(host=host, user=user, passwd=pw, database=db)
+
+#executing the mysql query to filter out the rows
+c = conn.cursor()
+c.execute("select * from disease__nodes;")
+data = c.fetchall()
+column_names = [desc[0] for desc in c.description]
+#converting the table obtained to a dataframe
+df = pd.DataFrame(data, columns= column_names)
+#print (df)
+
+#sending the api request
+def post_request(term, p):
+
```

Comment:
@NayanikaEL I recommend you to add docstrings to each function so that the reviewer is clear about what the function is doing. Below is a good example of a docstring

```
def add_binary(a, b):
    '''
    Returns the sum of two decimal numbers in binary digits.

            Parameters:
                    a (int): A decimal integer
                    b (int): Another decimal integer

            Returns:
                    binary_sum (str): Binary string of the sum of a and b
    '''
    binary_sum = bin(a+b)[2:]
    return binary_sum
```